### DISCUSSION TITLE
getContinuation on replies

### DISCUSSION THREAD

**Discussion by EricRodabaugh** [LuanRT/YouTube.js#634] [Q&A] [answered]
The current implementation of what I've been doing:
1. Get the base comments
2. Check to see if there are any replies
3. If there are call getReplies on the base comment thread
4. Check to see if has_continuation on the comment thread that is returned from getReplies
5. If there are call getContinuation on the comment thread that is returned from getReplies

This has been working like a charm the past few months, but now all of a sudden the getContinuation (in step 5) isn't returning any of the next page of comments. I have updated to 9.2.1 and wasn't sure if something changed and I missed it or something else is going on.

Thanks!

**Comment by LuanRT** [comment, marked as answer]
Looks like we forgot to update the code [here](https://github.com/LuanRT/YouTube.js/blob/0c2cdc1599d3fc2ed1f8578e0683478e0bde46d7/src/parser/classes/comments/CommentThread.ts#L93), so that's indeed a bug. Thanks for reporting!
